pkg/queue/manager_test.go

```diff
@@ -533,7 +532,10 @@ func TestStatus(t *testing.T) {
 		}
 	}
 	for _, wl := range workloads {
-		_ = manager.AddOrUpdateWorkload(&wl)
+		// We ignore the ErrClusterQueueDoesNotExist since we never set up ClusterQueue in this test, and the error should be occurred.
+		if err := manager.AddOrUpdateWorkload(&wl); err != nil && !errors.Is(err, ErrClusterQueueDoesNotExist) {
+			t.Fatalf("Failed to add or update workloads: %v", err)
+		}
```

Comment:
I think this change is very good and super safe, if you extract to separate PR we can merge immediately

Reply:
Opened: https://github.com/kubernetes-sigs/kueue/pull/3838